<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="S0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>choose a method from the drop-down</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="strike">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>and press &quot;select Choice&quot;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>before filling in the parameters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rf">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>lastly, press &quot;Calculate!&quot;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wildcard1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="wildcard2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="opt_type">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="output">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="choice">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Binomial Tree</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monte Carlo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Crank-Nicolson</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="choose">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Choice</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
